nntrainer/graph/network_graph.h

```diff
@@ -191,19 +191,26 @@ class NetworkGraph {
   sharedConstTensors forwarding(sharedConstTensors input);
 
   /**
-   * @brief     backwarding network graph
-   * @param[in] input data
-   * @param[in] iteration
+   * @brief     getter of ordered graph
+   * @retval    ordered LayerNode list
    */
-  void backwarding(sharedConstTensors input, int iteration);
+  std::vector<LayerNode> getSorted() { return Sorted; }
```

Comment:
`const std::vector<LayerNode> &` to prevent copy but maintain value semantics?

Reply:
done.